src/main/java/com/paneedah/mwc/entities/BanditEntityFactory.java

```diff
@@ -42,7 +43,8 @@ public void createEntity(ModContext modContext) {
         .withTexturedModelVariant("com.paneedah.mwc.models.BanditNew5", "banditnew5.png")
         .withHurtSound("hurt")
         .withAmbientSound("drawweapon")
-//        .withStepSound("step")
+        .withStepSound("step")
+           // not sure when step sound was excluded, i re added so they can't sneak up 
```

Comment:
This is probably actually broken so revert this please

Reply:
done!